game/scripts/vscripts/Sections/Chapter4/SectionOpening.ts

```diff
@@ -231,6 +234,9 @@ function onStop() {
         MinimapEvent(DotaTeam.GOODGUYS, playerHero, Vector(0, 0, 0).x, Vector(0, 0, 0).y, MinimapEventType.TUTORIAL_TASK_FINISHED, 0.1)
     }
 
+    if (failedScanDialogToken)
+        dg.stop(failedScanDialogToken)
```

Comment:
set `failedScanDialogToken` to undefined here so we don't call this again (shouldn't cause issues since we don't reuse tokens right now though)